runtime.md

```diff
@@ -2,22 +2,26 @@
 
 ## State
 
-The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
-The runtime state is stored in a JSON encoded file.
-It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
-On Linux based systems the state information should be stored in `/run/opencontainer/containers`.
-The directory structure for a container is `/run/opencontainer/containers/<containerID>/state.json`.
-By providing a default location that container state is stored external applications can find all containers running on a system.
-
-* **`version`** (string) Version of the OCI specification used when creating the container.
-* **`id`** (string) ID is the container's ID.
-* **`pid`** (int) Pid is the ID of the main process within the container.
-* **`bundlePath`** (string) BundlePath is the path to the container's bundle directory.
-
+Metadata for containers are stored on disk so that external tools can consume and act on this information.
+It is recommended that this data be stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the metadata MUST be stored in `/run/opencontainer/containers`.
+For non-Linux based systems the location of the root metadata directory is currently undefined.
+Within that directory there MUST be one directory for each container created, where the name of the directory MUST be the ID of the container.
+For example: for a container with an ID of `173975398351`, there will be a corresponding directory: `/run/opencontainer/containers/173975398351`.
```

Comment:
While we're tightening up this text, it might be time to revisit my [container-ID charset suggestions][1].  The spec still reads (to me) like you could setup a container whose ID was `../../../etc/passwd`.

[1]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/P9gZBYhiqDE